Dockerfile.build

```diff
@@ -0,0 +1,20 @@
+ARG DISTRO='quay.io/centos/centos:stream8'
+FROM $DISTRO
+
+ARG JENKINS_HOME
+
+ARG JENKINS_USER=jenkins
+ARG JENKINS_USER_ID=1110
+ARG JENKINS_USER_GID=100
+
+
+SHELL ["bash", "-x", "-c"]
+
+COPY ./ ./
+ARG FOR_MAKE_CHECK=1
+RUN source run-make-check.sh && prepare
+RUN dnf clean all && rm -rf /var/cache/dnf/*
```

Comment:
You're right.

In Docker I think they (recently?) added the [`--mount=cache` option to the `RUN` command](https://docs.docker.com/engine/reference/builder/#example-cache-apt-packages) in order to keep an external caching layer for these kind of commands.

Reply:
Interesting. Thanks for the pointer. I think this will be also in recent podman versions, but I have not tested it. I definitely want the containerfile and supporting bits to also work with podman, even if docker is used on the jenkins builders.